process/ReleaseProcess.md

```diff
@@ -10,6 +32,53 @@ The focus of this document refers to generating stable releases of Zowe. For ble
+
+Distributions that increment the version number represent a change in the support relationship that a customer using Zowe or a vendor extending Zowe has with the underlying Zowe distribution.  Each Zowe version is associated with a conformance program that determines the contract an extender of Zowe has with the base Zowe distribution.  If an extension has achieved the criteria for Zowe version *n* conformance, then that extension will continue to work unmodified with all subsequent release or modification distributions within the same version n. Version n+1 will have new conformance criteria and a vendor product conformance with version *n* may need to be modified to  continue to work with *n+1*. 
+
+### How many active distributions at a given time?
+
+There will be only one Active LTS distribution at a given time. The dates of when current distribution switches to Active LTS and to Maintenance distribution will vary depending on the judgment of the Zowe community. The open community will be given at least 30 days notice of intent to declare and Active LTS or Maintenance distribution. 
+
+A new current distribution that increments the version number of Zowe will be declared when there is a change impacting consumers of Zowe that the community needs.
+- feedback on a new architectural direction that deprecates the prior version and/or 
+- consumers of Zowe are likely to need time to develop, change and test and plan for implementation of software in advance of the next Active LTS distribution 
+- a new conformance program for extenders to be made aware of any changes that they may need to accommodate based on the scope of new, deprecated or removed function
+
+### Compatibility
+
+Once an Active LTS is declared, it is the community's intent to maintain compatibility for APIs, CLI plug-ins and Application Framework applications within the same version. The intent is for any extensions that have achieved Zowe conformance status for the Active LTS version, to run without changes from one release in a version to the next. 
+
+In current phase only, a new Zowe version may introduce a change that will limit prior extensions or cause them to become inoperable without the extension requiring modification.  This could be in the form of documentation, scripts or code requiring the extending offering to update their software. The Zowe community expects this to be a rare occurrence. If it does occur, the required changes to the extension will be communicated during a Current Release and Zowe extenders and consumers will be given at least 90 day notice of the required change.  All extensions who have Zowe conformance will be explicitly contacted and other communication will be made through mailing lists, slack, and other public channels. 
```

Comment:
If we are in the current phase of a release, then is a conformance program active? I believe it should not be, and we should not have a strict 90 day notification period of breaking changes while in 'current' development. The strict notification period should instead be 90 days before going `Active LTS`, at which point conformance would begin for that major version. At any other point in 'current development', notice is best effort through regular communication channels.

Reply:
@armstro Hi Bruce, could you look at this comment? Thanks!

Reply:
@MarkAckert @RASakach Mark I agree conformance does not apply to "current" releases - I am not sure what is stated where to clarify this so adding Rose here (or @TaylorDonner or @Tbr00ksy ).....it might just be we place a statement in the conformance criteria that it only applies to Active LTS (and Maintenance LTS) rather than try to describe in Support Policy in Release Docs?

Reply:
@MarkAckert @armstro, FYI:  @TaylorDonner, @Tbr00ksy :  As the Onboarding Squad works through the Conformance Updates for LTS Release, we are facing several decisions: 

1. Naming standards for the badging and
2. Consider rewording SECTION 6 PARAGRAPH 2 of the Conformance T&Cs which currently reads: 

- “_When a new version of the Test Evaluation Guide is released, conformant applications and services must demonstrate that they pass the updated tests and meet the updated evaluation criteria within 60 days. Applications that do not do so within the allotted time must not be marked as conformant thereafter. A previously-Qualifying Offering can re-conform to the new Test Evaluation Guide to regain conformance at any time by following the steps described above.”_

To revise as follows:  “_When a new version of the Test Evaluation Guide is released, conformant applications and services must demonstrate that they pass the updated tests and meet the updated evaluation criteria **to earn the badge associated with the "new version"**. Applications that do not do so will **retain their prior conformant badge**. A previously-Qualifying Offering can re-conform to the new Test Evaluation Guide **to gain "current"** conformance at any time by following the steps described above.”_

Current plan is to present a proposal for the Badge naming and revisions to T&Cs to ZLC.  If this is the appropriate path we can move forward on the next ZLC meeting.